web/packages/design/src/StepSlider/StepSlider.story.tsx

```diff
@@ -27,9 +27,11 @@ export default {
   title: 'Design/StepSlider',
 };
 
-const singleFlow = { default: [Body1, Body2] };
+const singleFlow = { default: [Body1, Body2, Body3] };
```

Comment:
I took the liberty of adding another step. Without it demonstrating that wrapping works wasn't quite as clear as I'd like as going forward or back moved you to the same step. ;)

I also made the buttons in Body2 and Body3 consistent with Body1. The Next button is always the primary button.